docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |find a person by tags |locate people easily
+
+|`* * *` |user |sort people by name |locate people easily
+
+|`* * *` |user |set a list of favourite people|
```

Comment:
In this case it would be better to add a `So that I can` field here on why there should be a list of favourite people?